builder-support/debian/wforce-trackalert.postinst

```diff
@@ -17,7 +17,7 @@ case "$1" in
       echo "done"
     fi
     echo -n "Modifying trackalert.conf to replace password and key..."
-    SETKEY=`echo "makeKey()" | wforce | grep setKey`
+    SETKEY=`echo "makeKey()" | wforce -c | grep setKey`
```

Comment:
```suggestion
    SETKEY=`echo "makeKey()" | trackalert -c | grep setKey`
```

Reply:
```trackalert -c``` needs trackalert.service to be running. While installing trackalert we can assume it's not running
```suggestion
    SETKEY=`echo "makeKey()" | trackalert | grep setKey`
```

Reply:
yes this change would work